diff --git hbase-shell/src/main/ruby/shell/commands/alter.rb hbase-shell/src/main/ruby/shell/commands/alter.rb
index 18cafa8..2c3aa6f 100644
--- hbase-shell/src/main/ruby/shell/commands/alter.rb
+++ hbase-shell/src/main/ruby/shell/commands/alter.rb
@@ -49,7 +49,7 @@ To delete the 'f1' column family in table 'ns1:t1', use one of:
   hbase> alter 'ns1:t1', 'delete' => 'f1'
 
 You can also change table-scope attributes like MAX_FILESIZE, READONLY, 
-MEMSTORE_FLUSHSIZE, DEFERRED_LOG_FLUSH, etc. These can be put at the end;
+MEMSTORE_FLUSHSIZE, DURABILITY, etc. These can be put at the end;
 for example, to change the max size of a region to 128MB, do:
 
   hbase> alter 't1', MAX_FILESIZE => '134217728'
